test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceProjectTests.cs

```diff
@@ -49,7 +49,7 @@ public LegacyPackageReferenceProjectTests(DispatcherThreadFixture fixture)
             using (var testDirectory = TestDirectory.Create())
             {
                 var testBaseIntermediateOutputPath = Path.Combine(testDirectory, "obj");
-                TestDirectory.Create(testBaseIntermediateOutputPath);
+                Directory.CreateDirectory(testBaseIntermediateOutputPath);
```

Comment:
Can we still make these go through the TestDirectory class so it's easier to keep track of folders that are being created via tests?

Reply:
It's a sub folder of a test folder, I don't see a reason to track it independently. Calling TestDirectory.Create was wrong imo

Reply:
oh. as long as the parent directory was created using testdirectory, works for me.